src/Compilers/CSharp/Test/Syntax/Syntax/SyntaxNormalizerTests.cs

```diff
@@ -940,7 +940,7 @@ private static void TestNormalizeStatement(string text, string expected)
         {
             var node = SyntaxFactory.ParseStatement(text.NormalizeLineEndings());
             var actual = node.NormalizeWhitespace("  ").ToFullString();
-            Assert.Equal(expected.NormalizeLineEndings(), actual.NormalizeLineEndings());
+            AssertEx.Equal(expected.NormalizeLineEndings(), actual.NormalizeLineEndings());
```

Comment:
Produces nicer error messages when strings are wrong (like a diff of how they're wrong).